templates/security/esm.html

```diff
@@ -10,632 +10,424 @@
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/ebff661c-glusterfs+logo.png",
-        alt="Gluster",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/2f883d00-go+logo.png",
-        alt="GoLang",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/fabdefb7-kafka+logo.png",
-        alt="Kafka",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/d00a6ac1-nagios+logo.png",
-        alt="Nagios",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/2dfa0b07-nginex+logo.png",
-        alt="nginex",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/2303838b-node-logo.png",
-        alt="Node J S",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
-      </div>
-      <div class="col-small-1 col-medium-1 col-1">
-        {{ image (
-        url="https://assets.ubuntu.com/v1/c9baeb67-openjdk+logo.png",
-        alt="Open J D K",
-        attrs={"class":"p-logo-section__logo"},
-        width="280",
-        height="280",
-        hi_def=False,
-        loading="lazy",
-        ) | safe
-        }}
+{% block content %}
+  {% call(slot) vf_hero(
+    title_text='Expanded Security Maintenance <br />for Ubuntu and open source',
+    layout='50/50'
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Extend the lifetime of your favorite Linux and the open source you use on top with reliable security maintenance for up to 12 years.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/security/esm#get-in-touch" class="p-button--positive">Speak to an expert</a>
```

Comment:
in case JS is disabled this link won't open a modal. We need a fallback that will still open a Contact Us static page in case JS is disabled.